bugbug/db.py

```diff
@@ -119,7 +119,7 @@ def last_modified(path):
     last_modified = utils.get_last_modified(url)
 
     if last_modified is None:
-        raise Exception("Last-Modified is not available")
+        raise LastModifiedNotAvailable("Last-Modified is not available")
```

Comment:
The name of the class says it all, we can pass an empty message now:
```suggestion
        raise LastModifiedNotAvailable()
```